app/assets/javascripts/controllers/projects_controller.js

```diff
@@ -4,6 +4,10 @@ angular.module("OpenHq").controller("ProjectsController", function($scope, Users
   $scope.projects = projects;
   $scope.teamUsers = UsersRepository.all();
 
+  Project.query({archived: true}).then(function(resp){
+    $scope.archived_count = resp.length;
```

Comment:
I think with angular the main style would be using camel case variables, so that’s what I’ve tried to do throughout